apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx

```diff
@@ -324,22 +100,14 @@ export const GlobalAppsContainer: React.FC = () => {
     return (
         <>
             {appUrls.map((url, index) => (
-                <GlobalIframe
-                    key={`${get(user, 'id', null)}-${url}`}
-                    pageUrl={url}
-                    ref={el => iframeRefs.current[index] = el}
+                <IFrame
+                    key={url}
+                    src={url}
+                    reloadScope='user'
```

Comment:
what if I need to reload the global miniapp when changing organization?

Reply:
Like completely reload?
When we should probably add some mark in schema, and use it here